packages/contracts/src/init/LibInitWheel.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { getUniqueEntity } from "@latticexyz/world/src/modules/uniqueentity/getUniqueEntity.sol";
+
+import { Wheel, DefaultWheel, Name, WheelData } from "../codegen/Tables.sol";
+
+library LibInitWheel {
+  error GuisePrototypeInitSystem__InvalidSkill();
+
+  function init() internal {
+    bytes32 wheelEntity = add(
+      "Wheel of Attainment",
+      WheelData({ totalIdentityRequired: 0, charges: 3, isIsolated: false })
+    );
+
+    DefaultWheel.set(wheelEntity);
+
+    add("Wheel of Isolation", WheelData({ totalIdentityRequired: 128, charges: 4, isIsolated: true }));
+  }
+
+  function add(string memory name, WheelData memory wheelData) internal returns (bytes32 entity) {
```

Comment:
I'd prefer `_add` and make it `private` instead of `internal`
(I know you're just copying my code here, so this is me criticizing myself)